webapp/sources/rudder/rudder-rest/src/test/resources/api/api_groups.yml

```diff
@@ -1122,6 +1122,37 @@ response:
       }
     }
 ---
+description: Update a node group query with invalid query comparator value
+method: POST
+url: /api/latest/groups/00000000-cb9d-4f7b-abda-ca38c5d643ea
+headers:
+  - "Content-Type: application/json"
+body: >-
+  {
+    "id": "00000000-cb9d-4f7b-abda-ca38c5d643ea",
+    "query": {
+      "select": "nodeAndPolicyServer",
+      "composition": "and",
+      "transform": "invert",
```

Comment:
why invert ? (really to know: it's not the default queries, and not relevant to that problem, so I'm wondering)

Reply:
it's to have it at least covered in test somewhere, I don't think the value is tested anywhere.
I think the value for "transform" is validated before the criteria